### PR TITLE
[realestate.com.au] New ruleset

### PR DESCRIPTION
**Pull request by youdly**

### CODE REVIEW

File: src/chrome/content/rules/realestate.com.au.xml [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+	<target host="snappass.partner-platform.realestate.com.au" />
+	<target host="pay.realestate.com.au" />
+	<target host="www.pay.realestate.com.au" />
+	<target host="pexa.realestate.com.au" />
+	<target host="pki.realestate.com.au" />
+	<target host="propertyplatform.realestate.com.au" />
+	<target host="propertyplatform-demo.realestate.com.au" />
+	<target host="app-news-wordpress.rca-prod.realestate.com.au" />
+	<target host="rca-news-ui.rca-prod.realestate.com.au" />
+	<target host="stargazing.reaio.realestate.com.au" />
```
Comment (J0WI) [inline src/chrome/content/rules/realestate.com.au.xml]: This host is gone

Comment (youdly) [reply src/chrome/content/rules/realestate.com.au.xml]: Removed

**Comment by J0WI** [comment]
Some tests are still failing

**Comment by youdly** [comment]
Removed invalid hosts.

**Comment by youdly** [comment]
Removed more invalid hosts and squashed commits.

**Pull request merged**